types/src/system/auction/mod.rs

```diff
@@ -52,10 +52,9 @@ pub type UnbondingPurses = BTreeMap<AccountHash, Vec<UnbondingPurse>>;
 pub trait Auction:
     StorageProvider + SystemProvider + RuntimeProvider + MintProvider + AccountProvider + Sized
 {
-    /// Returns era_validators.
+    /// Returns era validators for each configured trusted eras.
```

Comment:
/// Returns active validators and auction winners for a number of future eras determined by the configured auction_delay.